tools/lib/api_tests.py

```diff
@@ -96,10 +97,16 @@ def remove_subscriptions(client):
 def render_message(client):
     # type: (Client) -> None
 
-    request = dict(content='**foo**')
+    # {code_example|start}
+    # Render a message
+    request = {
+        'content': '**foo**'
+    }
     result = client.render_message(request)
-    assert result['result'] == 'success'
-    assert result['rendered'] == '<p><strong>foo</strong></p>'
+    # {code_example|end}
+
+    fixture = TEST_FUNCTIONS['render-message']['fixture']
```

Comment:
I think the `mypy` error is because `TEST_FUNCTIONS` is defined later in the file, and is not declared with `global`.  I'd just move the definition up, probably?